radix-engine-interface/src/blueprints/resource/access_rules.rs

```diff
@@ -159,14 +157,8 @@ impl AccessRulesConfig {
             .unwrap_or_else(|| self.get_default_mutability())
     }
 
-    fn get_default_mutability(&self) -> AccessRule {
-        match &self.default_auth_mutability {
-            AccessRuleEntry::AccessRule(access_rule) => access_rule.clone(),
-            AccessRuleEntry::Group(name) => match self.grouped_auth.get(name) {
-                Some(access_rule) => access_rule.clone(),
-                None => AccessRule::DenyAll,
-            },
-        }
+    pub fn get_default_mutability(&self) -> AccessRule {
+        self.resolve_entry(&self.default_auth_mutability)
```

Comment:
Wow, was surprised rust didn't complain about multiple `self` reference use but realized that these are read-only refs. Magic!